src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from '@/components/Tabs/Tabs.module.scss';
```

Comment:
Can this file be named `index.tsx`?
That way, we can import this component as `import Tabs from '@/components/Tabs';` (instead of `import Tabs from '@/components/Tabs/Tabs';`)